driver/src/price_finding/naive_solver.rs

```diff
@@ -108,6 +122,7 @@ impl PriceFinding for NaiveSolver {
         let mut exec_buy_amount: Vec<u128> = vec![0; orders.len()];
         let mut exec_sell_amount: Vec<u128> = vec![0; orders.len()];
 
+        let mut matching_orders_indices: Option<(usize, usize)> = None;
```

Comment:
ubernit: this could be a `[usize; 2]` and slightly simplify the code where you iterate over it.